java/integTest/src/test/java/glide/SharedClientTests.java

```diff
@@ -88,6 +88,7 @@ private static Stream<Arguments> clientAndDataSize() {
     @MethodSource("clientAndDataSize")
     public void client_can_handle_concurrent_workload(BaseClient client, int valueSize) {
         ExecutorService executorService = Executors.newCachedThreadPool();
+        @SuppressWarnings("rawtypes")
```

Comment:
same qustion

Reply:
I updated code here and change suppression to be the same as above.
Meanwhile, a good read https://stackoverflow.com/a/9542131/21176342